services/QuillLMS/app/models/auth_credential.rb

```diff
@@ -41,6 +45,6 @@ def refresh_token_valid?
   end
 
   def refresh_token_expires_at
```

Comment:
Note, I was looking for uses of this method since this is now a general method with a provider specific implementation, and there's a different calculation for [refresh_token_expires_at](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/controllers/api/v1/users_controller.rb#L14) in the API controller.

Reply:
I saw that one.  At some point I would like to refactor all of the refresh_token stuff but I think we can leave as-is for now since this will just return nil for `refresh_token_expires_at` since the `auth_credential.refresh_token` conditional will return false for clever login.

Reply:
I looked at this code yesterday and decided it wasn't relevant.  On second pass today, as far as I can tell, those keys aren't used anymore.